CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(CMAKE_CXX_STANDARD 11)
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Is this needed?  Note that 3.12 here concerns the cmake version and has nothing to do with Python version.

Reply:
reverted